src/lib/entity.js

```diff
@@ -602,9 +602,14 @@ export function printEntity(entity, onDoubleClick) {
  * @return {Element} Entity created
  */
 export function createEntity(definition, cb) {
+  const re = /^[\'a-zA-Z0-9 _-]{1,50}$/;
```

Comment:
The updated regex is here:
https://github.com/uwblueprint/cdc-backend/blob/ade67c8949c59279205dbf12466fee6bcad6fd87/app/library/api_schemas.py#L10
Can you update this line to match that please?